src/components/Card/Card.jsx

```diff
@@ -0,0 +1,62 @@
+import React, {useState} from "react";
+
+import Toggler from "../Toggler/Toggler";
+import CardStatus from "../CardStatus/CardStatus";
+import AddCardBtn from "../AddCardBtn/AddCardBtn";
+
+import {
+    CardContainer,
+    CardImageWrapper,
+    CardImage,
+    CardBody,
+    CardContent,
+    CardName,
+    CardPoints,
+    Wrapper
+} from "./Card.style";
+
+const Card = ({imageUrl, title, points, status, pageName}) => {
+    const [selected, setSelected] = useState(false);
+
+    const toggle = () => setSelected(!selected);
+
+    return (
+        <CardContainer>
+            <CardImageWrapper>
+                <CardImage src={imageUrl} />
+            </CardImageWrapper>
+            <CardBody>
+                <CardContent>
+                    <CardName>
+                        {title}
+                    </CardName>
+                    <CardPoints>
+                        {points} баллов
+                    </CardPoints>
+                </CardContent>
+                {
+                    pageName === 'awards' &&
+                    <Wrapper onClick={toggle}>
+                        <Toggler selected={selected} />
+                    </Wrapper>
+                    ||
+                    pageName === '/' &&
+                    <Wrapper onClick={toggle}>
+                        <Toggler selected={selected} />
+                    </Wrapper>
+                    ||
+                    /* change to proper page name*/
+                    pageName === '/prevday' &&
```

Comment:
Looks like you wont be able to check previous day by url, because url will be the same for any day - `/`.
I suggest to pass to the "Togller" component a property like status.
Inside Toggler you could check the status props and render appropriate UI elements.
Like:
```
if(status === "planning") {
    return <div>Planing button</div>
}
```

Ps If you accept this approach create an enum for statuses)